docs/pages/_document.tsx

```diff
@@ -36,7 +36,6 @@ export default class MyDocument extends Document<{ css?: string }> {
       <Html lang="en">
         <Head>
           <script src="/static/libs/tippy/tippy.all.min.js" />
-          <script src="/static/libs/nprogress/nprogress.js" />
```

Comment:
We should also be able to delete the file itself along with its css file

Reply:
Done!